hikari/impl/entity_factory.py

```diff
@@ -1466,6 +1467,21 @@ def deserialize_guild_private_thread(
             thread_created_at=thread_created_at,
         )
 
+    def deserialize_stage_instance(self, payload: data_binding.JSONObject) -> stage_instances.StageInstance:
+        guild_scheduled_event_id = (
+            snowflakes.Snowflake(payload["guild_scheduled_event_id"]) if payload["guild_scheduled_event_id"] else None
+        )
```

Comment:
```suggestion
        raw_event_id = payload["guild_scheduled_event_id"]
        guild_scheduled_event_id = snowflakes.Snowflake(raw_event_id) if raw_event_id else None
```

Would make this a bit cleaner